index.html

```diff
@@ -1463,6 +1463,15 @@ <h5>
             <p its-locale-filter-list="en" lang="en">Ellipses are used to indicate a truncation of text, an unfinished sentence or a break in speech. An ellipsis in Chinese consists of six dots, takes up the space of two Hanzi characters, and is horizontally and vertically centered within its character frame. This is normally achieved using two <span class="uname" translate="no">U+2026 HORIZONTAL ELLIPSIS</span> characters, side-by-side.
             <p its-locale-filter-list="zh-hans" lang="zh-hans">省略号又作删节号，表示节略原文、语句未完或语气的不连续。删节号呈现上占两个汉字空间、包含六个省略点且在水平和垂直方向均位于字面正中，通常使用两个连续的<span class="uname" translate="no">U+2026 HORIZONTAL ELLIPSIS […]</span>来实现。</p>
             <p its-locale-filter-list="zh-hant" lang="zh-hant" class="checkme">刪節號又作省略號，表示節略原文、語句未完或語氣的不連續。刪節號呈現上佔兩個漢字空間、包含六個居中的刪節點且在水平和垂直方向均位於字面正中，通常使用兩個連續的<span class="uname" translate="no">U+2026 HORIZONTAL ELLIPSIS […]</span>來實現。</p>
+            <div class="note" id="u22ef">
+              <p its-locale-filter-list="en" lang="en">Chapter 6.2 of v14 of the Unicode Standard suggests using <span class="uname" translate="no">U+22EF MIDLINE HORIZONTAL ELLIPSIS</span> [⋯] for ellipses.</p>
+              <p its-locale-filter-list="zh-hans" lang="zh-hans">在Unicode标准第14版的6.2章中，也推荐使用<span class="uname" translate="no">U+22EF MIDLINE HORIZONTAL ELLIPSIS</span> [⋯]作为省略号。</p>
+              <p its-locale-filter-list="zh-hant" lang="zh-hant">在Unicode標準第14版的6.2章中，亦推薦使用<span class="uname" translate="no">U+22EF MIDLINE HORIZONTAL ELLIPSIS</span> [⋯]作為刪節號。</p>
+              
+              <p its-locale-filter-list="en" lang="en">This code point, with appropriate rotatation and replacement mechanism, will be centered centered within its character frame regardless of whether it is in vertical or horizontal writing mode, which is more in line with the Chinese typesetting requirements. However, this character is classified as a mathematical symbol in Unicode and may not be rendered correctly on some early operating systems.</p>
```

Comment:
typo: centered centered

I thought the problem had more to do with font support than with OS support.  Also, it may not actually be very relevant to mention its classification as mathematical, since the Unicode Standard is clearly nowadays suggesting non-mathematical usage.  Probably enough to say that some (perhaps older) fonts may not support it.